src/platform/packages/private/kbn-esql-editor/src/overwrite.scss

```diff
@@ -1,39 +0,0 @@
-/* Editor styles for any layout mode */
-/* NOTE: Much of this is overriding Monaco styles so the specificity is intentional */
-
-.ESQLEditor .monaco-editor .monaco-hover {
-  display: block !important;
-}
-
-.ESQLEditor .monaco-editor .margin-view-overlays .line-numbers {
-  color: $euiColorDisabledText;
-}
-
-// Currently focused line(s)
-.ESQLEditor .monaco-editor .current-line ~ .line-numbers {
-  color: $euiTextSubduedColor;
-}
-
-// Suggest (autocomplete) menu
-.ESQLEditor .monaco-editor .suggest-widget, .ESQLEditor .monaco-editor .suggest-details-container {
-  @include euiBottomShadow;
-  border-radius: $euiBorderRadius;
-}
-
-.ESQLEditor .monaco-editor .suggest-details-container {
-  background-color: $euiColorEmptyShade;
-  line-height: 1.5rem;
-}
-
-.ESQLEditor_errorMessage {
-  @include euiTextBreakWord;
```

Comment:
Looks like this one needs added back. From the same link Tim provides above, you can find this solution: 
![CleanShot 2025-01-23 at 07 25 34@2x](https://github.com/user-attachments/assets/f9bfc4c8-e2ef-4fdd-a9d0-db8b3c3a10b0)

Reply:
Bur in case I miss something I added it bach here https://github.com/elastic/kibana/pull/208003/commits/5f23fd6520f925870bb1e8f4a35c6750cefd7ae5